src/AppEntry.tsx

```diff
@@ -74,13 +73,30 @@ const AppWithConfig: React.FunctionComponent = () => {
 // eslint-disable-next-line react/display-name
 const AppEntry: React.FunctionComponent = React.memo(() => (
   <Provider store={init(logger).getStore()}>
-    <I18nextProvider i18n={appServicesi18n}>
+    <I18nProvider
+      lng={'en'}
+      resources={{
+        en: {
```

Comment:
We can do that when we need to do that I suppose?

Reply:
Yes, what matters is that the we have externalized all the strings at this point, so that if we do want to internationalize we aren't retrofitting entire codebases. The actual loading of other languages we can do later.